redis/Cargo.toml

```diff
@@ -10,8 +10,36 @@ repository = "https://github.com/djc/bb8"
 [dependencies]
 async-trait = "0.1"
 bb8 = { version = "0.8", path = "../bb8" }
-redis = { version = "0.24", default-features = false, features = ["tokio-comp"] }
+redis = { version = "0.24" }
+# redis = { version = "0.24", default-features = false, features = [
+#   "tokio-comp",
+# ] }
 
 [dev-dependencies]
 futures-util = "0.3.15"
 tokio = { version = "1.0.0", features = ["macros", "rt-multi-thread"] }
+
+[features]
+default = ["redis/default"]
+acl = []
```

Comment:
Any features added here that don't enable anything in the upstream redis crate just don't do anything, so we should not add them (or, I guess you meant `acl = ["redis/acl"]`?).

Reply:
Yes, I meant the latter - I wasn't cautious. Change made!